vmdb/app/controllers/ems_container_controller.rb

```diff
@@ -0,0 +1,23 @@
+class EmsContainerController < ApplicationController
+  include EmsCommon        # common methods for EmsInfra/Cloud/Container controllers
+  include ContainersCommonMixin
+
+  before_filter :check_privileges
```

Comment:
I have added access to some of the features in db/fixtures/miq_product_features.yml but as you start adding more buttons you will need a feature with same id in miq_product_features.yml. By default super admins have access to everything as long as feature is defined in above mentioned yml.